devices/sonoff_t1_uk_3gang_v1.1.rst

```diff
@@ -295,7 +295,7 @@ them in one area, and simply pass that ID later on. For example, above you can s
 output being created with the ID ``blue_led`` for the blue LED. Later on it is then transformed
 into a :doc:`monochromatic light </components/light/monochromatic>`.
 If you additionally want the buttons to control the relays, look at `the complete Sonoff T1 UK 3 Gang
-with automation example <https://github.com/OttoWinter/esphomedocs/blob/current/devices/sonoff_t1_uk_3gang_v1.1.yaml>`__.
+with automation example <https://github.com/esphome/esphomedocs/blob/current/devices/sonoff_t1_uk_3gang_v1.1.yaml>`__.
```

Comment:
Since you are changing it anyway, it may as well point to the current repo name. =)
```suggestion
with automation example <https://github.com/esphome/esphome-docs/blob/current/devices/sonoff_t1_uk_3gang_v1.1.yaml>`__.
```